components/dhcp.yml

```diff
@@ -20,3 +20,4 @@ rules:
 - package_dhcp_removed
 - service_dhcpd_disabled
 - sysconfig_networking_bootproto_ifcfg
+- package_kea_removed
```

Comment:
Does the kea package replace the dhcp package? Is the dhcp package present in RHEL 10? If the dhcp package isn't present in RHEL 10 we should remove the rule package_sshd_removed from RHEL 10 content.

Reply:
It does, I have removed `package_dhcp_removed` from all profiles in RHEL 10.